iblsorter/main.py

```diff
@@ -74,10 +78,20 @@ def run(
 
     ctx = Context(ctx_path)
     ctx.params = params
+    ctx.motion_params = motion_params
```

Comment:
Here I probably would have added the new parameters groups in a Nested structure. 
This is possible with Pydantic and we would have something like:

params:
- ... old params
- preprocessing
- registration